src/flamenco/vm/Local.mk

```diff
@@ -6,4 +6,6 @@ $(call make-bin,fd_vm_tool,fd_vm_tool,fd_flamenco fd_ballet fd_util)
 endif
 
 $(call make-unit-test,test_vm_interp,test_vm_interp,fd_flamenco fd_ballet fd_util)
-$(call run-unit-test,test_vm_interp)
+
+# Todo(lheeger-jump): enable this
```

Comment:
What's the issue with this?

Reply:
This test fails but Liam is going to fix that in the runtime branch.